plugin-procfs/README.md

```diff
@@ -1,3 +1,138 @@
+|`swap_cached`|Gauge|Bytes|Memory that was placed in swap and brought back to RAM, but remains in swap file|LocalMachine|LocalMachine||
+|`active`|Gauge|Bytes|Memory that has been used recently|LocalMachine|LocalMachine||
+|`inactive`|Gauge|Bytes|Used memory that can be reclaimed, freed, or moved to swap.|LocalMachine|LocalMachine||
+|`mapped`|Gauge|Bytes|Memory used for files associated with processes in memory, file and memory mapping.|LocalMachine|LocalMachine||
+
+### Attributes
+
+#### Kind
+
+The kind of the memory is the allocated memory space reserved by the system or the hardware :
+
+|Value|Description|
+|-----|-----------|
+|`resident`|Physical memory actually used by a process|
+|`shared`|Part of memory used simultaneously by several processes|
+|`virtual`|Total virtual memory address space requested by a process|
+
+The kind of the CPU time delta is the average CPU time spent by various tasks :
+
+|Value|Description|
+|-----|-----------|
+|`user`|CPU time used to run userspace processes (not Kernel system processes)|
+|`system`|CPU time used by the system (Kernel)|
+|`guest`|CPU time used by a guest virtual machine when a hypervisor virtualizes environments|
+
+#### cpu_state
+
+The CPU states correspond to all categories of time during which the CPU performs various tasks :
+
+|Value|Description|
+|-----|-----------|
+|`user`|CPU time used to run userspace processes (not Kernel system processes)|
+|`nice`|CPU time consumed by tasks that give way to other higher priority processes|
+|`system`|CPU time used by the system (Kernel)|
+|`idle`|Time the CPU is in sleep mode or waiting for tasks|
+|`irq`|CPU time spent handling hardware interrupts (input/output or peripheral signals)|
+|`softirq`|CPU time spent handling software interrupts (delayed interrupts scheduled by the kernel)|
+|`steal`|Time a hypervisor uses CPU for other virtual machines on the same host|
+|`guest`|CPU time used by a guest virtual machine when a hypervisor virtualizes environments|
+|`guest_nice`|CPU time used by a guest virtual machine for processes with "nice" priority|
+
+## Configuration
+
+Here is a configuration example of the plugin. It's part of the ALUMET configuration file (eg: `alumet-config.toml`).
+
+```rust
+[plugins.procfs]
+poll_interval = "1s"
+flush_interval = "5s"
+```
+
+To active the plugin to collect metrics relative to kernel utilization :
+
+```rust
+[plugins.procfs.kernel]
+enabled = true
+poll_interval = "5s"
+```
+
+Also, you can collect or not more precise metrics about the memory consumption with each detail level :
+
+```rust
+[plugins.procfs.memory]
+enabled = true
+poll_interval = "5s"
+metrics = [
+    "MemTotal",
+    "MemFree",
+    "MemAvailable",
+    "Cached",
+    "SwapCached",
+    "Active",
+    "Inactive",
+    "Mapped",
+]
+```
+
+You can active collect of processes metrics for the plugin with 2 different strategies (system watcher or internal event) :
```

Comment:
And what does it change?
Imagine you're a new user, that has never used Alumet nor the plugin. You would ask yourself: What are the differences between the strategies? Why do they exist? What should I choose?